django_redis/client/default.py

```diff
@@ -266,46 +266,43 @@ def get(
 
     def persist(
         self, key: Any, version: Optional[int] = None, client: Optional[Redis] = None
-    ) -> None:
+    ) -> bool:
         if client is None:
             client = self.get_client(write=True)
 
         key = self.make_key(key, version=version)
 
-        if client.exists(key):
-            client.persist(key)
+        return client.persist(key)
 
     def expire(
         self,
         key: Any,
         timeout,
         version: Optional[int] = None,
         client: Optional[Redis] = None,
-    ) -> None:
+    ) -> bool:
         if client is None:
             client = self.get_client(write=True)
 
         key = self.make_key(key, version=version)
 
-        if client.exists(key):
-            client.expire(key, timeout)
+        return client.expire(key, timeout)
 
-    def pexpire(self, key, timeout, version=None, client=None):
+    def pexpire(self, key, timeout, version=None, client=None) -> int:
```

Comment:
why does this return an integer?

`PERSIST`, `EXPIRE` and `PEXPIRE` all return 1 or 0, why only two of them return booleans?

Reply:
Good question, I noticed it but forgot to look into the why.
Did so now :) Looking at redis documentation they all return int.
But redis-py parses the response according to the existing in `RESPONSE_CALLBACK` as seen here:
https://github.com/redis/redis-py/blob/396e7323959b7a79d72602afee3ef648dc23adf9/redis/client.py#L619-L624

It appears that `PEXPIRE` is simply not listed there, resulting in the 'raw' integer response.

Reply:
maybe then parsing it to boolean is better so that we keep the same interface, I will raise the issue to `redis-py` team. Can you also add `pexpire_at` since you are at it?